gadget/ondisk.go

```diff
@@ -385,10 +376,10 @@ func UpdatePartitionList(dl *OnDiskVolume) error {
 
 // CreatedDuringInstall returns a list of partitions created during the
 // install process.
-func CreatedDuringInstall(layout *OnDiskVolume) (created []string) {
+func CreatedDuringInstall(lv *LaidOutVolume, layout *OnDiskVolume) (created []string) {
```

Comment:
I think I had this question before, but the code has changed so much,  can't this function actually live under gadget/install?  same now for WasCreatedDuringInstall ?

Reply:
probably it can, I did not really look to change where things are located much and just retrofitted what was there to work the way we wanted it, but I will now look into moving things around to more appropriate locations

Reply:
We can probably do this in a followup